ls1intum/Artemis
# Store Apollon Diagrams per course

### Checklist

### Motivation and Context


https://github.com/ls1intum/Artemis/issues/1250

### Description
Apollon Diagrams now have a course assigned to them and only the diagrams that are part of the course will show up when looking for the diagrams in a course.

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Management
3. Navigate to Exercises
4. Click on Create Drag and Drop Model Quiz
5. Create diagrams in different courses and see if they are visible in courses other than the one they were created in. They shouldn't be visible.
-> old diagrams are not visible because there are so few on live that it isn't needed to put in an extra request for diagrams with no courseId 
6. Open a diagram and generate a quiz exercise from it.

### Screenshots


![Bildschirmfoto von 2020-05-02 19-15-55](https://user-images.githubusercontent.com/57408107/80870875-73e57280-8ca9-11ea-8b4c-27cfbce54005.png)
